packages/generic/wirisplugin-generic.src.js

```diff
@@ -91,25 +91,27 @@ export default class GenericIntegration extends IntegrationModel {
   /**
    * Object containing all the Telemeter functions.
    */
-  telemeter = {
+  static Telemeter = {
```

Comment:
Why are we converting this field to `static`? On top of that, renaming it may cause retro compatibility issues, I think.